docs/text/rolling.rst

```diff
@@ -21,6 +21,11 @@ the last 10 steps.
 A rolling mechanism creates such time series for every time step by creating sub time series of the sensor data of the
 last 10 time steps.
 
+<!---
+The above scenario cannot be solved by tsfresh, can it?
```

Comment:
@TKlerx yes, you are totally correct, this can not be handled by the methods in our package.